cpp/src/platform/windows/LogImpl.cpp

```diff
@@ -41,7 +41,7 @@ errno_t fopen_s(FILE** pFile, const char *filename, const char *mode)
 {
     if (!pFile)
     {
-        _set_errno(EINVAL);
```

Comment:
set_errno was introduced for platforms that need to do "something special" with the errno assignment (local threat variable handling etc). I'd just prefer to use that even though the posix standard says you can assign to directly to the variable.

> Seems that MINGW doesn't define msvcrt version

Hence why I said:

> if you can figure out the correct defines for MINGW